community/portworx/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 name: portworx
-version: 1.0.19
+version: 1.0.20
 description: A Helm chart for installing Portworx on Kubernetes.
 tillerVersion: ">=2.7.2"
 kubeVersion: ">=1.10.0"
-appVersion: "2.5.3"
+appVersion: "2.5.5"
```

Comment:
previous version update pr is merged https://github.com/IBM/charts/pull/818
can you rebase this against master